### ISSUE TITLE
Cannot run program on Mac

### ISSUE THREAD

**Issue by KyngK** [arpruss/raspberryjammod#62]
I was following the instructable (https://www.instructables.com/Python-coding-for-Minecraft/), and it says if the error with "cannot run program" comes up, then you should add python to your path. I added python to my path, but when running my script it comes up with an error that says "Error java.io.IOException: Cannot run program "python" (in directory "mcpipy"): error=2, No such file or directory". I know the script is there, since when I try to run a script name that isn't there it says "Cannot find script". Is there something else I need to do for it to work?

I'm using version 1.8.9

**Comment by arpruss** [comment]
Do you have python installed on your machine? (Try it in Terminal.) If so, go to the mod's options and set the directory for the python binary.

**Comment by KyngK** [comment]
I do have python installed - I also changed python3 to python in my path. Where are the mod's options?

**Comment by KyngK** [comment]
> Do you have python installed on your machine? (Try it in Terminal.) If so, go to the mod's options and set the directory for the python binary.


I do. Where are the mod options?

**Comment by arpruss** [comment]
In the game menu.

**Comment by KyngK** [comment]
> In the game menu.

That worked, thanks! One quick question:
Where is the documentation for the api? It says it's at mcpipy.com but that link doesn't exist

**Comment by arpruss** [comment]
You can probably find the old website on the way back machine at archive.org
Does the example code, e.g., donut.py, work?

**Comment by KyngK** [comment]
> You can probably find the old website on the way back machine at archive.org Does the example code, e.g., donut.py, work?

I found it - thank you!

> Does the example code, e.g., donut.py, work?

The donut.py does not work, since it is using the from mine import *, which the mine module is not defined (it needs to be mcpi). I tried fixing this but then more errors in the code came up from the actual mcpi module. I tried the benchmark.py file, however, and it worked with no modification.

By the way, thank you so much for this mod!! I've always wanted to program in Minecraft (without mcfunction/datapacks), and I know python fairly well, so this is super helpful to me (:
